src/Control/Algebra/Handler.hs

```diff
@@ -33,3 +36,12 @@ module Control.Algebra.Internal
 --
 -- @since 1.1.0.0
 type Handler ctx m n = forall x . ctx (m x) -> n (ctx x)
+
+-- | Composition of handlers.
+--
+-- @since 1.1.0.0
+(~<~) :: (Functor n, Functor ctx1) => Handler ctx1 m n -> Handler ctx2 l m -> Handler (Compose ctx1 ctx2) l n
+hdl1 ~<~ hdl2 = fmap Compose . hdl1 . fmap hdl2 . getCompose
+{-# INLINE (~<~) #-}
+
+infixr 1 ~<~
```

Comment:
This fixity & precedence were chosen to:

1. match those of `Control.Category.<<<`, and

2. be compatible with `.` s.t. you can compose handlers `foo . bar` and `quux . baz` straightforwardly as `foo . bar ~<~ quux . baz`, without any extra parens.